packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionModule.tests.js

```diff
@@ -161,32 +163,53 @@ QUnit.module('Mention format', () => {
         };
 
         const element = MentionFormat.create(data);
-        assert.strictEqual(element.innerText, '#John Smith', 'correct inner text');
+
+        new MentionFormat({}, element);
+        assert.strictEqual(element.children[0].innerText, '#John Smith', 'correct inner text');
     });
 
     test('Change default content renderer', function(assert) {
-        const data = {
+        const expectedData = {
             value: 'John Smith',
             marker: '@',
-            id: 'JohnSm',
+            id: 'JohnSm'
+        };
+        const data = {
+            ...expectedData,
             keyInTemplateStorage: 'my_key_in_storage'
         };
 
         MentionFormat.addTemplate({ marker: '@', editorKey: 'my_key_in_storage' }, {
             render: ({ container, model: mentionData }) => {
                 container.innerText = 'test';
-                assert.deepEqual(mentionData, data);
+                assert.deepEqual(mentionData, expectedData);
             }
         });
 
         let element = MentionFormat.create(data);
 
-        assert.strictEqual(element.innerText, 'test');
+        new MentionFormat({}, element);
+        assert.strictEqual(element.children[0].innerText, 'test');
 
         MentionFormat.removeTemplate({ marker: '@', editorKey: 'my_key_in_storage' });
         element = MentionFormat.create(data);
+        new MentionFormat({}, element);
+        assert.strictEqual(element.children[0].innerText, '@John Smith');
+    });
+
+    test('async template renders correctly in React strict mode (T1187274)', function(assert) {
```

Comment:
this test know nothing about react. I suggest use kind of name like this: "mention content should be rendered to <smth> container/place" because this test checked this behavior